README.md

```diff
@@ -41,29 +41,26 @@ Before you begin, make sure you have the following installed on your system:
     git clone https://github.com/citrineos/citrineos-core
     ```
 
-1. Navigate to the CitrineOS Server directory:
-
-    ```shell
-    cd citrineos-core/Server
-    ```
-
-1. Install project dependencies:
+1. Install project dependencies from root dir:
 
    ```shell
-   ./unix-init-install-all.sh
+   npm run install-all
    ```
 
-1. Start the server and its supporting infrastructure with:
+1. The docker container should be initialized by running `docker-compose -f ./Server/docker-compose.yml up -d` or 
```

Comment:
This command does not work. The yml file seems to be moved into root folder.

Reply:
yep, I just updated the readme